lib/core_to_mucore.ml

```diff
@@ -1513,7 +1604,8 @@ let normalise_fun_map_decl
             make_fun_with_spec_args
               (fun env st ->
                  let@ returned =
-                   Translate.return_type loc env st (ret_s, ret_ct) (accesses, ensures)
+                   (* TODO: add ghost return to frontend: https://github.com/rems-project/cn/issues/181 *)
+                   Translate.return_type loc env st (ret_s, ret_ct) (accesses, snd ensures)
```

Comment:
If it's not being tested or usable, please remove the ghost return code from this PR.